Workspace token updates (#644)
rudderlabs/rudderstack-docs: docs/rudderstack-open-source/installing-and-setting-up-rudderstack/kubernetes.mdx

If you're using the RudderStack-hosted control plane, follow these steps to set up the data plane on your Kubernetes cluster:

1. Log into your [RudderStack Open Source dashboard](https://app.rudderstack.com/signup?type=opensource).

> 2. Copy your workspace token by going to **Settings** > **Company**, as shown:

Reviewer: ```suggestion
2. Copy your workspace token from **Settings** > **Company**, as shown:
```
